docs/leaderboard/develop/api-reference/response-objects.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
I'm going to let it slide for now, but I think this page does not have value/is misleading. The main index page references this one stating it has specific information, but it is very generic.

Reply:
Yeah I copied this over from the other docs sections, but I agree that it's very generic right now. I can remove this page and/or the references to it if that seems like the better move